addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="vitotronic"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 org.eclipse.smarthome.thing-description.xsd">
+
+	<bridge-type id="bridge">
+		<label>Bridge Vitotronic Optolink adapter</label>
+		<description>This bridge represents the Vitotronic Optolink adapter 
+		</description>
+
+		<config-description>
+			<parameter name="ipAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>IP Address of the Optolink Adapter </label>
+				<description>The IP address of the Optolink adapter 
+				</description>
+				<default>127.0.0.0</default>

```

Comment:
is this a reasonable default? As you set required to true, there should actually be no default value available